WordPressKit/WordPressKit/TaxonomyServiceRemoteXMLRPC.m

```diff
@@ -102,6 +104,39 @@ - (void)createTag:(RemotePostTag *)tag
                          } failure:failure];
 }
 
+- (void)updateTag:(RemotePostTag *)tag
+          success:(nullable void (^)(RemotePostTag *tag))success
+          failure:(nullable void (^)(NSError *error))failure
+{
+    NSMutableDictionary *extraParameters = [NSMutableDictionary dictionary];
+    [extraParameters setObject:tag.tagID ?: [NSNull null] forKey:TaxonomyXMLRPCIDParameter];
```

Comment:
Sort of.  This does add the `term_id` but where it appears matters.  With this change, if you were to inspect `xmlrpcParameters` in `editTaxonomyWithType` you'd see something like:
```
(lldb) po xmlrpcParameters
<__NSArrayM 0x60400065bb70>(
0,
username,
password,
{
    term_id = 14;
    description = "<RemotePostTag: 0x60400065b750> Tag6[14]";
    name = Tag6;
    taxonomy = "post_tag";
}
)
```
However, the docs for the XML-RPC method `wp.editTerm` specify that the term ID should be passed after the password.  The params should look something like this:
```
(lldb) po xmlrpcParameters
<__NSArrayM 0x60400065bb70>(
0,
username,
password,
14,
{
    description = "<RemotePostTag: 0x60400065b750> Tag6[14]";
    name = Tag6;
    taxonomy = "post_tag";
}
)
```